graphql/src/entities/internal_reconcile_report.py

```diff
@@ -7,22 +7,84 @@
 import strawberry
 
 
+@strawberry.type  # Not strictly clean, but alternative is duplicating classes in graphql adapter.
+@dataclasses.dataclass
+class Phantom(pydantic.BaseModel):
+    # IMPORTANT: Whenever properties are added/removed/modified/renamed, update constructor.
+    # Python doesn't cap 32 bit/4 byte int size, but GraphQL can't handle larger ints.
```

Comment:
This does unfortunately mean that these values, which should be integers, will be returned with a decimal (1234.0). Might be a way to define a custom GraphQL type, but it wasn't straightforward.

Reply:
https://bugs.earthdata.nasa.gov/browse/ORCA-622